index.ts

```diff
@@ -283,6 +283,12 @@ commander
   .description('Display help')
   .action(help);
 
+/**
+ * Displays clasp version
+ */
+commander
+.version('1.3.1', '-v, --version');
```

Comment:
This number will never be kept in sync with the actual latest version. Can we bump versions in one place?

https://github.com/google/clasp/blob/master/package.json#L3
https://github.com/tj/commander.js/issues/48#issuecomment-49087390

Reply:
Good call, thanks 👍